Utilities/NanoAODReader.cc

```diff
@@ -54,9 +54,10 @@ class NanoAODReader {
   ~NanoAODReader() {
     cout << "NanoAODReader Destructed" << endl;
   }
+
   vector<string> GetFileNames() {
     vector<string> out;
-    string basepath = "/afs/cern.ch/work/s/siluo/wprime/filenames/";
+    string basepath = "filenames/";
```

Comment:
Here, the relative filepath allows for not changing all string basepaths in yet another file...